assets/scss/base/_table.scss

```diff
@@ -277,3 +277,121 @@ Styleguide Table.Marker
   height: 10px;
   border-radius: 50%;
 }
+
+/*
+Responsive
+
+Responsive Table
+
+Markup:
+<table>
+  <thead>
+    <tr class="table__row--desktop-headings">
+      <th>Heading 1</th>
+      <th>Heading 2</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr class="table__row--responsive">
+      <td class="table__detail--responsive" data-heading="Heading 1">
+        <p class="table__cell-content--responsive">Row 1 Detail 1<p>
+      </td>
+      <td class="table__detail--responsive" data-heading="Heading 2">
+        <p class="table__cell-content--responsive">Row 1 Detail 2</p>
+      </td>
+    </tr>
+    <tr class="table__row--responsive">
+      <td class="table__detail--responsive" data-heading="Heading 1">
+        <p class="table__cell-content--responsive">Row 2 Detail 1</p>
+      </td>
+      <td class="table__detail--responsive" data-heading="Heading 2">
+        <p class="table__cell-content--responsive">Row 2 Detail 2</p>
+      </td>
+    </tr>
+    <tr class="table__row--responsive">
+      <td class="table__detail--responsive" data-heading="Really quite a long heading for column 1">
+        <p class="table__cell-content--responsive">Row 3 Detail 1</p>
+      </td>
+      <td class="table__detail--responsive" data-heading="Really quite a long heading for column 2">
+        <p class="table__cell-content--responsive">Row 3 Detail 2</p>
+      </td>
+    </tr>
+    <tr class="table__row--responsive">
+      <td class="table__detail--responsive" data-heading="Heading 1">
+        <p class="table__cell-content--responsive">Row 4 Detail 1 - has really quite a lot of content to include.</p>
+      </td>
+      <td class="table__detail--responsive" data-heading="Heading 2">
+        <p class="table__cell-content--responsive">Row 4 Detail 2 - has really quite a lot of content to include also.</p>
+      </td>
+    </tr>
+    <tr class="table__row--responsive">
+      <td class="table__detail--responsive" data-heading="Heading 1">
+        <div class="table__cell-content--responsive">
+          <p>Row 5 Detail 1 - paragraph One</p>
+          <p>Row 5 Detail 1 - paragraph Two</p>
+          <p>Row 5 Detail 1 - paragraph Three</p>
+        </div>
+      </td>
+      <td class="table__detail--responsive" data-heading="Heading 2">
+        <div class="table__cell-content--responsive">
+          <p>Row 5 Detail 2 - paragraph One</p>
+          <p>Row 5 Detail 2 - paragraph Two</p>
+          <p>Row 5 Detail 2 - paragraph Three</p>
+        </div>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+Styleguide Table.Responsive
+*/
+
+@media (max-width: 639px) {
```

Comment:
not a massive fan of magic numbers, can you align to the way it is done elsewhere and use the `media` include and the defined `mobile`, `tablet` or `desktop` https://github.com/hmrc/assets-frontend/blob/3610585baeb96bb4ad9d73d20cfb73864e095b50/assets/govuk_elements/public/sass/elements/_forms.scss#L150